nselib/httpcookies.lua

```diff
@@ -81,13 +81,13 @@ CookieJar = {
     setmetatable(o, self)
     self.__index = self
 
+    --Setting default values for the options
+    o.options.no_cookie_overwrite = o.options.no_cookie_overwrite or false
```

Comment:
The default behavior should not overwrite the cookies defined by the user automatically.